packages/core/admin/admin/src/hooks/useInjectReducer/useInjectReducer.js

```diff
@@ -0,0 +1,10 @@
+import { useEffect } from 'react';
+import { useStore } from 'react-redux';
+
+export function useInjectReducer(namespace, reducer) {
```

Comment:
JSDOC! When you use it, why to use it and the usual imo. Don't think it's elaborate enough to need to go in the contributor docs?

Reply:
JSDoc ✅ 

> Don't think it's elaborate enough to need to go in the contributor docs?

I'd make this dependent on the follow-up exploration. If we start to use this hook throughout the codebase as e.g. a decoupling mechanism, I'd add it to the contrib docs. For now I agree and would keep it out of it.